integ/components/deadline/deadline_04_workerFleetHttps/test/deadline_04_workerFleetHttps.test.ts

```diff
@@ -84,6 +85,7 @@ describe.each(testCases)('Deadline WorkerFleet tests (%s)', (_, id) => {
       return awaitSsmCommand(bastionId, params);
     }
     else {
+      console.error(`Did not find a secrect ARN for ${testingStackName}`);
       return;
```

Comment:
It probably warrants throwing an exception here since these tests are only run for HTTPS, right?

Reply:
Yeah, makes sense, will change it.

Reply:
Looks good